introduction.mdx

```diff
@@ -21,11 +21,11 @@ Upstash offers developers serverless databases and messaging platforms to build
 ## Concepts
 
 <CardGroup cols={2}>
-  <Card title="Serverless" href="/concepts/serverless">
+  <Card title="Serverless" href="common/concepts/serverless">
     Upstash is serverless. You don't need to provision any infrastructure. Just
     create a database and start using it.
   </Card>
-  <Card title="Scaling to Zero" href="/concenpts/scale-to-zero">
+  <Card title="Scaling to Zero" href="common/concepts/scale-to-zero">
```

Comment:
```suggestion
  <Card title="Scaling to Zero" href="/common/concepts/scale-to-zero">
```
please always use an absolute path
otherwise things will break horribly if we ever decide to move pages around